### DISCUSSION TITLE
load table not possible

### DISCUSSION THREAD

**Discussion by Roland27801** [JvanKatwijk/qt-dab#338] [Q&A]
Working on Raspberry Pi 400
Version 6.8 compiled on the Raspi
I give my coordinates to the program. Also the the .txdate.tii File is in the home directory.
All old transmitters are shown, also on the map. But we have many new transmitters here in northwestern Germany. So I want to load a new table with new transmitters. Using "load table" with the result in german: "Laden der Datenbank gescheitert", on the console: 'cannot load libtii-lib.so' . 

Is there anything I can do to get an actual Transmitterlist?

best regards
Roland

**Comment by JvanKatwijk** [comment]
Your message seems truncated to me.
Anywat, do you have a database installed?

Op di 26 nov 2024 om 23:01 schreef Roland27801 ***@***.***>:

> I give my coordinates to the program. Also the
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/338>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQAKJOETIQCCVEMCHNT2CTVT7AVCNFSM6AAAAABSROEX2GVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGU3DCMRXG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Jan van Katwijk

**Comment by andimik** [comment]
Me too (in the mail). 

Jan, if you click on "view it on GitHub" you will see the full text. Seems he was too fast and submitted it too early (= mail was sent) and changed the text later online only.

**Comment by JvanKatwijk** [comment]
The most recent database database is to be found here
https://www.dropbox.com/scl/fi/6kc6mj12mulh421yf0has/txdata.tii?rlkey=61t55h3t4u61phdk7mzajuswy&dl=0

as usual, change the name in ".txdata.tii: and put it in the home directory

iIt might well be that the copy in the sourcetree is somewhat older


Op wo 27 nov 2024 om 15:09 schreef andimik ***@***.***>:

> Me too (in the mail).
>
> Jan, if you click on "view it on GitHub" you will see the full text. Seems
> he was too fast and submitted it too early (= mail was sent) and changed
> the text later online only.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/338#discussioncomment-11395192>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQDZIPWKXCBRMAQWQ4T2CXHBNAVCNFSM6AAAAABSROEX2GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMZZGUYTSMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Jan van Katwijk

**Comment by davidbstanley** [comment]
I am having a similar issue.  In my case, the most recent database is successfully downloaded by the qt-db-loader app.  Clicking 'refresh table' in the Configuration window works but the updated table does not persist after restarting qtdab.  Clicking refresh table updates it but this has to be done every time qtdab is restarted.  I am using the later qt6-DAB 6.9.6 on Windows 10.  I tried running as administrator but that did not help.

**Reply by gvanem** [reply]
This silly "libtii-lib" library is to protect/maskerade the real source of this DATA-list.
It's at: https://www.dablist.org/qtdab/qtdab_dabtx_data.php

It would be trivial to have Qt download this and cache it as needed.

**Comment by JvanKatwijk** [comment]
If you switch off the "local DB" setting, the software will load - upon
program startup - the database stored in your home directory.
The "local DB" switch was just added for making life easier on systems
where no app exist to load a freesh copy of the database


Op zo 25 jan 2026 om 10:27 schreef davidbstanley ***@***.***>:

> I am having a similar issue. In my case, the most recent database is
> successfully downloaded by the qt-db-loader app. Clicking 'refresh table'
> in the Configuration window works but the updated table does not persist
> after restarting qtdab. Clicking refresh table updates it but this has to
> be done every time qtdab is restarted. I am using the later qt6-DAB 6.9.6
> on Windows 10. I tried running as administrator but that did not help.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/338#discussioncomment-15596572>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQB5DNADC5J3NILJAH34ISD6ZAVCNFSM6AAAAABSROEX2GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTKNJZGY2TOMQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Jan van Katwijk

**Reply by davidbstanley** [reply]
Thanks.  I don't know how I missed that!  All OK now.

**Comment by JvanKatwijk** [comment]
No problem
Feel free to ask


Op zo 25 jan 2026 om 12:10 schreef davidbstanley ***@***.***>:

> Thanks. I don't know how I missed that! All OK now.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/338#discussioncomment-15597048>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQH5JG3X34OBYGJ4BQ34ISQBZAVCNFSM6AAAAABSROEX2GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTKNJZG4YDIOA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Jan van Katwijk

**Comment by JvanKatwijk** [comment]
Absolutely true, bit the feature was asked for


Op ma 2 feb 2026 om 10:32 schreef Gisle Vanem ***@***.***>:

> This silly "libtii-lib" library is to protect/maskerade the real source of
> this DATA-list.
> It's at: https://www.dablist.org/qtdab/qtdab_dabtx_data.php
>
> It would be trivial to have Qt download this and cache it as needed.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/JvanKatwijk/qt-dab/discussions/338#discussioncomment-15670676>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACCPHQCLCAR352WJE6NH6QD4J4KTRAVCNFSM6AAAAABSROEX2GVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTKNRXGA3DONQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Jan van Katwijk
